snap/snapcraft.yaml

```
@@ -44,12 +44,13 @@ parts:
     source: .
     stage-packages: [ git, sqlite3, openssh-client ]
     build-packages: [ git, libpam0g-dev, libsqlite3-dev, build-essential]
-    build-snaps: [ go, node/18/stable ]
+    build-snaps: [ go/latest/stable, node/18/stable ]
```

```suggestion
    build-snaps: [ go/1.21/stable, node/18/stable ]
```

given that we lock go every where else, we should do here as well.